sales_tax.rb

```diff
@@ -1,9 +1,74 @@
-class Product
+require_relative("print_receipt.rb")
+
+$exmpt = ["chocolates","book","pills","chocolate"]
+$products_arr = []
 
-  def initialize(item_name, price, exempt, imported)
+class Product
+  attr_accessor :qty, :item_name, :price, :exempt, :imported, :tax
+  def initialize(qty, item_name, price, exempt, imported, tax)
+    @qty = qty
     @item_name = item_name
-    @price = price
+    @price = price.to_f
     @exempt = exempt
     @imported = imported
+    @tax = tax
+  end
+
+  # Calculate the different taxes :-
+  def calculate_tax
+    @tax = 0
+    if @exempt == false && @imported == true
+      @tax = @price * (0.05 + 0.10)
```

Comment:
If you have initialized `attr_accessor` then you should use that instead of instance variables.

Reply:
If you have initialized `attr_accessor` then you should use that instead of instance variables.